app/static/js/controllers/admin.organizations.controller.js

```diff
@@ -114,22 +114,23 @@ angular.module('linc.admin.organizations.controller', [])
         modalScope.dataSending = true;
         $scope.LincApiServices.Organizations({'method': 'post', 'data': data}).then(function(response){
           $scope.Notification.success({
-            title: 'Organization Info', 
+            title: 'Organization Info',
             message: 'New Organization successfully created',
-            position: "right", 
-            duration: 2000 
+            position: "right",
+            duration: 2000
           });
           var organization = response.data;
           organization.created_at = (organization.created_at || "").substring(0,19);
           organization.updated_at = (organization.updated_at || "").substring(0,19);
           organization.selected = true;
           $scope.$parent.organizations.push(organization);
           $scope.Selecteds.push(organization);
+          $scope.$parent.OrganizationsUpdated();
```

Comment:
This change, calling an existing method that was previously already called for editing and deleting, ensures that adding will also trigger the update. This ends up running some code that is not strictly necessary for adding an org, since, for example, adding a new org won't require an existing imageset to be reassigned, but it seems simpler code-wise to have one single method to call when various organization updates need to be performed.